src/dotnet/commands/dotnet-new/NewCommandShim.cs

```diff
@@ -24,7 +27,16 @@ internal class NewCommandShim
 
         public static int Run(string[] args)
         {
-            return New3Command.Run(CommandName, CreateHost(), new TelemetryLogger(null), FirstRun, args);
+            var sessionId = Environment.GetEnvironmentVariable(MSBuildForwardingApp.TelemetrySessionIdEnvironmentVariableName);
+            var telemetry = new Telemetry(new NuGetCacheSentinel(new CliFallbackFolderPathCalculator()), sessionId);
```

Comment:
@livarcocc this was copied from the MSBuild one, I'll take a look to see if we can reuse that one all up